help/tips/2.txt

```diff
@@ -0,0 +1,2 @@
+<<You can buy or sell items from players shops.
```

Comment:
no, they can not be 0X because manaplus does loadData(num + 1); and loadData(num - 1);

yes, I will put the tips in a separate commit